clang/tools/clang-scan-deps/ClangScanDeps.cpp

```diff
@@ -472,7 +465,9 @@ class FullDeps {
             JOS.attributeArray("command-line",
                                toJSONStrings(JOS, MD.getBuildArguments()));
             JOS.attribute("context-hash", StringRef(MD.ID.ContextHash));
-            JOS.attributeArray("file-deps", toJSONSorted(JOS, MD.FileDeps));
+            JOS.attributeArray("file-deps", [&] {
```

Comment:
I removed the sorting here to avoid having to allocate all the paths and store them into a container.